Tensile/KernelWriterAssembly.py

```diff
@@ -225,16 +225,18 @@ def getCompileArgs(self, sourceFileName, objectFileName, *moreArgs, isa=None, wa
     if wavefrontSize is None:
       wavefrontSize = self.kernel["WavefrontSize"]
 
-    archHasV3 = globalParameters["AsmCaps"][isa]["HasCodeObjectV3"]
-
     launcher = shlex.split(os.environ.get('Tensile_ASM_COMPILER_LAUNCHER', ''))
 
     rv = launcher + [globalParameters['AssemblerPath'],
           '-x', 'assembler',
           '-target', 'amdgcn-amd-amdhsa']
 
-    if archHasV3:
-      rv += ['-mcode-object-version=2' if globalParameters["CodeObjectVersion"] == "V2" else '-mcode-object-version=4']
```

Comment:
Why -mcode-object-version was 4 for V3 before change?
Is 3 the correct value?

Reply:
No, 3 means COV3, we just had V4 partially hacked in before